apps/dashboard/src/pages/questionnaire-page.tsx

```diff
@@ -0,0 +1,14 @@
+import { PageMeta } from '@/components/page-meta';
+import { AuthCard } from '../components/auth/auth-card';
+import { QuestionnaireForm } from '../components/auth/questionnaire-form';
+
+export function QuestionnairePage() {
+  return (
+    <>
+      <PageMeta title="Setup your workspace" />
+      <AuthCard>
+        <QuestionnaireForm />
```

Comment:
The CTA button colors are not aligned with the design, all steps should use the black variant.
<img width="1250" alt="Screenshot 2024-11-25 at 23 31 17" src="https://github.com/user-attachments/assets/8426bde8-27e9-4917-ae64-c88c2c1cd173">

Reply:
@LetItRock I couldn't find any button with a dark variant, should I create one?

Reply:
it should be a default variant